pixeltable/utils/arrow.py

```diff
@@ -0,0 +1,112 @@
+import pyarrow as pa
+from typing import Any, Dict, Iterator, Iterable, Optional, Union, List
+import pixeltable.type_system as ts
+import pathlib
+import logging
+
+_logger = logging.getLogger(__name__)
+
+_pa_to_pt : Dict[pa.DataType, ts.ColumnType] = {
+        pa.string() : ts.StringType(nullable=True),
+        pa.timestamp('us') : ts.TimestampType(nullable=True),
```

Comment:
What happens if the importer sees a timestamp with a different resolution?

Reply:
Currently that would trigger an exception, at the moment we are the only users. My thinking is by using it we will gather ideas. (also relates to larger discussion about how to break this. interface up in my first comment)